src/main/resources/templates/sparql/person/educationAndTrainingIsDateSuppressed.sparql

```diff
@@ -0,0 +1,11 @@
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+PREFIX vivo: <http://vivoweb.org/ontology/core#>
+
+CONSTRUCT {
+    ?dateTimeInterval vivo:supplementalInformation ?isDateSuppressedWithId .
+} WHERE {
+    <{{uri}}> obo:RO_0000056 ?training .
+    ?training vivo:dateTimeInterval ?dateTimeInterval .
+    ?dateTimeInterval vivo:supplementalInformation ?isDateSuppressed .
+    BIND( CONCAT( STR(?isDateSuppressed), REPLACE(STR(?training), "(^.*/)", "::") ) AS ?isDateSuppressedWithId ) .
+}
```

Comment:
Missing trailing new line.

Reply:
At glance this seems rather common so I am going to instead have this a suggestion rather than a change request.